packages/web-app/web-app-front/src/components/Spinner/index.ts

```diff
@@ -0,0 +1 @@
+export { Spinner } from './Spinner';
```

Comment:
I don't see why we should reexport files from index.ts. Please delete these re-exporting index files.